### PR TITLE
Report if an object has details in browseMode

### PR DESCRIPTION
**Pull request by seanbudd**
### Link to issue number:

None

### Summary of the issue:

#12364 added a command to read the summary of the aria-details. To know whether or not an object has details, this should be reported. However, new users might not want to use this feature so #12409 added a config setting to opt-in.

### Description of how this pull request fixes the issue:

Discards the "has details" state change message unless the annotations config option is turned on.

### Testing strategy:

Chrome system tests have been added.

Manual testing should be done in firefox using the test sample. 

### Known issues with pull request:

None

### Change log entries:
New features (under annotations)

- adds an option in advanced to report if an object has details in browse mode. 

### Code Review Checklist:

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual tests.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.

### CODE REVIEW

File: tests/system/robot/chromeTests.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def test_aria_details():
 	actualSpeech = _chrome.getSpeechAfterKey('downArrow')
 	_asserts.strings_match(
 		actualSpeech,
-		"The word  marked content  cat  out of marked content  has a comment tied to it."
+		"The word  marked content  has details  cat  out of marked content  has a comment tied to it."
```
Comment (seanbudd) [inline tests/system/robot/chromeTests.py:72]: should this be reported after or before "cat"?

Comment (codeofdusk) [reply tests/system/robot/chromeTests.py:72]: The "has details" should be outside (i.e. reported before or after) the marked content, so the scope of the mark is clear.

Comment (feerrenrut) [reply tests/system/robot/chromeTests.py:72]: I think this is fine. Personally I think that "marked content has details" reads better than "has details marked content". Might be good to get a few more opinions.

**Comment by XLTechie** [comment]
Reef Turner wrote:

> I think this is fine. Personally I think that "marked content has details" reads better than "has details marked content". Might be good to get a few more
> opinions.

Agreed. Imo the former flows more smoothly.

**Review by feerrenrut — APPROVED** [review]
Looks good to me.

**Pull request merged**